app/css/app.css

```diff
@@ -1,8 +1,23 @@
+@import 'FiraSans/font.css';
```

Comment:
For now "compiled" app.css file will contain everything. Right now it's just 2.5kb gziped, so it sounds like we don't need to have several css bundles.